### DISCUSSION TITLE
Rear Cargo Door

### DISCUSSION THREAD

**Discussion by Viper787** [magknight/787aviatorsTracker#574] [Q&A] [answered]
**Describe the bug**
Clicking on the space defined to Open/Close rear cargo door not working.

**To Reproduce**
Click on area to open door, and it will not work

**Expected behavior**
Door should Open/Close

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac 10.12.6
 - X-Plane version 11.50r3
 - Version 1.68beta2

**Additional context**

**Comment by aeroniemi** [comment]
Are you sure you are using a new (v5) fuselage livery?

**Comment by Viper787** [comment]
> Are you sure you are using a new (v5) fuselage livery?

Just checked, yes that was the problem. Thx

**Comment by aeroniemi** [comment, marked as answer]
Rear/bulk cargo doors can only be opened in V5 liveries - you'll need to use one of those for it to open
